masonry/examples/calc.rs

```diff
@@ -10,6 +10,7 @@
 use std::sync::Arc;
 
 use accesskit::{DefaultActionVerb, Role};
+use dpi::LogicalSize;
```

Comment:
If our users need to use this, we should re-export this entire crate

Reply:
Done!